tests/baselines/reference/jsDeclarationsParameterTagReusesInputNodeInEmit1.errors.txt

```diff
@@ -14,17 +16,29 @@ tests/cases/conformance/jsdoc/declarations/file.js(8,1): error TS9006: Declarati
     
     module.exports = BaseFactory;
     
-==== tests/cases/conformance/jsdoc/declarations/file.js (1 errors) ====
+==== tests/cases/conformance/jsdoc/declarations/file.js (3 errors) ====
     /** @typedef {import('./base')} BaseFactory */
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+!!! error TS4081: Exported type alias 'BaseFactory' has or is using private name 'Base'.

```

Comment:
Do you think we should add a `Exported type alias 'BaseFactory' has or is using private name 'Base' from module "tests/cases/conformance/jsdoc/declarations/base".` formatted error?

Reply:
In the general case, yes -- in this case, with the `import('./base')` sitting right there, it's not so valuable.